.github/workflows/windows_test_comment_trigger.yml

```diff
@@ -42,14 +42,22 @@ jobs:
     outputs:
       PR_SHA: ${{ steps.sha.outputs.result }}
 
+  test_matrix:
+    name: Generate Test Matrix
```

Comment:
We don't need to run this job for all comments - only for those which trigger windows tests.
Please, add `needs: [windows_test_preparation]`.